### ISSUE TITLE
NVDA in Chrome fails to announce aria-describedby content if it includes inline HTML

### ISSUE THREAD

**Issue by terrill** [nvaccess/nvda#10813]
### Steps to reproduce:

Create a `<button>` element with a valid `aria-describedby` attribute. Inside the container referenced by `aria-describedby`, add content that includes inline HTML such as `<strong>`, `<em>` or `<span>` elements.  I've created a [demo page](http://terrillthompson.com/tests/aria/aria-describedby-inline.html) with several examples.   

### Actual behavior:

NVDA announces some, but not all, content referenced by aria-describedby: 

- It announces any text that is not wrapped in an inline HTML element. 
- It announces any text that is wrapped in a `<span>` element.  
- It does **not** announce any text that is wrapped in a `<strong>` or `<em>` element. 

### Expected behavior:

NVDA should announce all content referenced by aria-describedby, including content that is wrapped in  a `<strong>` or `<em>` element. 

#### NVDA installed/portable/running from source:

Installed 

#### NVDA version:

2019.3.1 

#### Windows version:

Windows 10 Home 

#### Name and version of other software in use when reproducing the issue:

Chrome 80

#### Does the issue still occur after restarting your PC?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No. However, I've tried this same version of NVDA in Firefox 73.0.1.  This is not an issue in Firefox; only in Chrome.  I've also tried JAWS 2020 in both Firefox and Chrome - it doesn't exhibit this behavior in either browser.

**Comment by Adriani90** [comment]
cc: @feerrenrut is there any work for Google Chrome planned during next releases? I guess this should get more priority since Edge Chromium will also benefit from it.

**Comment by Adriani90** [comment]
cc: @aleventhal since this seems to occur only in Crome.

**Comment by Adriani90** [comment]
Actually, after testing further, it seems this works as expected in all browsers, including Google Chrome 81 with NVDA Version: alpha-20015,56a62097.

@terrill please feel free to comment if you still see this issues with the software versions in my comment. We can reopen if this is still the case.
I am closing this for now as works for me.

**Comment by valenfv** [comment]
Hello, is anyone facing the same issue? I'm using NVDA v2020.4 on Chrome V94.0 on Windows Server 2016.
Thank you 😊
